neptune-bulk-loader/update-document-terms.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-underscore-dangle */
+import { z } from 'zod';
+import * as osClient from '../lib/open-search-client';
+
+const getTerms = async (
+  esUrl: string,
+  title: string,
+  contents: string
+) => osClient.percolateDocumentFields(
+  esUrl,
+  {
+    title,
+    contents,
+  }
+);
+
+const hitSchema = z.object({
+  _id: z.string(),
+  _source: z.object({
+    _bitstreamText: z.string().optional(),
+    dc_title: z.string(),
+  }),
+});
+
+const updateDocumentTerms = async (
+  esUrl: string,
+  index: string,
+  rawHit: unknown
+): Promise<void> => {
+  const hit = hitSchema.parse(rawHit);
+
+  if (hit._source._bitstreamText === undefined) return;
```

Comment:
We still want to percolate a document even if it doesn't have text from a PDF. We just send the title.

Reply:
Updated in dbe9d70